tests/distributions/test_transform.py

```diff
@@ -281,6 +281,18 @@ def test_ordered():
     vals = get_values(tr.ordered, Vector(R, 3), pt.vector, floatX(np.zeros(3)))
     assert_array_equal(np.diff(vals) >= 0, True)
 
+    # Check that positive=True creates positive and still ordered values
```

Comment:
Do we have a test that covers the forward/baward are inverses + jacobian that we could parametrize with the new parameters?